examples/graph_prediction/README.md

```diff
@@ -0,0 +1,105 @@
+# Example Code for Graph Prediction using GraphStorm
+Graph prediction, such as classification and regression, is a common task in Graph Machine
+Learning (GML) across various domains, including life sciences and chemistry. In graph prediction,
+the entire graph data is typically organized in a batch of subgraphs format, where each subgraph's
+nodes have edges only within the subgraph and no edges connecting to nodes in other subgraphs. GML
+labels are linked to these subgraphs. And then GML model makes prediction on new subgraphs as well.
+
+A typical operation used in graph prediction is called `Read-out`, e.g., `sum`, `mean`, `max` or
+`min`, which aggregates the representations of nodes in a subgraph to form one representation for
+the subgraph. Then outputs of the `Read-out` will be used to perform downstream prediction computation.
+
+The current version of GraphStorm can not directly perform graph prediction. But as GraphStorm
+supports node-level prediction, we can use a method called `super-node` to fulfill the graph 
+prediction requirement. Instead of using the `Read-out` operation, we can add a new node, called 
```

Comment:
```suggestion
supports node-level prediction, we can use a method called `super-node` to perform graph-level 
predictions. Instead of using the `Read-out` operation, we can add a new node, called 
```